ports/geckolib/tools/regen_style_structs.sh

```diff
@@ -65,6 +65,7 @@ fi
   -no-type-renaming                                                 \
   -DMOZILLA_INTERNAL_API                                            \
   -DMOZ_STYLO_BINDINGS=1                                            \
+  -DJS_DEBUG=1                                                      \
```

Comment:
Hm, can you explain what this is about? Is this just to make things compatible with the -DDEBUG passed below?

Ideally we'd generate both debug and release variants, and let the build system decide which to use based on whether we're doing a debug or release build of geckolib. Can you think of an easy way to do that?

Long-term, we'll probably want to generate the bindings at compile-time. But that will require geckolib to live in mozilla-central, so let's not worry about that for now.